### ISSUE TITLE
Reformatting callable.(arg, :arg) to callable.()(arg, :arg)

### ISSUE THREAD

**Issue by sholden** [samphippen/rubyfmt#229]
* Ruby version: ruby 2.7.1p83 (2020-03-31 revision a0c7c23c9c) [x86_64-darwin19]
* Rubyfmt git sha: 3435618222b247105c585730664be021cd601523

## Input file

```ruby
model = Object.new
callable = ->(model, *attrs) { puts("😱") }
callable.(model, :attr)
```

## Rubyfmt's output

```
model = Object.new
callable = -> (model, *attrs) { puts("😱") }
callable.()(model, :attr)
```

## This breaks because

rubyfmt adds an empty call with no params before adding the call with params.

**Comment by reese** [comment]
This is fixed in #343 (specifically 531bc4890397d6da0c0802a445717c71c47d0e5f), so the output on master is now the following:

```ruby
model = Object.new
callable = -> (model, *attrs) { puts("😱") }
callable.(model, :attr)
```

**Comment by sholden** [comment]
Yay! Amazing work on the big upsteam merge!
